index.js

```diff
@@ -51,6 +57,7 @@ function init() {
     ui.metricPerTile = metricPerTile
     canvas.updateTilesFromMetrics(metricPerTile, sumMetrics)
   })
+  ui.setUnsavedChangesCallback(() => canvas.getGrid().checkForEdits())
```

Comment:
I think I need the anonymous function to make sure 'this' is the correct context, no?